src/Microsoft.DotNet.Cli.Utils/BuiltInCommand.cs

```diff
@@ -100,6 +108,29 @@ public ICommand OnErrorLine(Action<string> handler)
             return this;
         }
 
+        private class BuiltInCommandEnvironment : IBuiltInCommandEnvironment
```

Comment:
It is going to have Get/Set CurrentDirectory once https://github.com/dotnet/cli/pull/2792 is merged.